integration/test/performance_tracker/performance_metrics_test.go

```diff
@@ -0,0 +1,46 @@
+//go:build linux && integration
+// +build linux,integration
+
+package data_collector
+
+import(
+	"testing"
+	"time"
+	"log"
+
+	"github.com/aws/amazon-cloudwatch-agent/integration/test"
+)
+
+const (
+	configPath = "resources/config.json"
+	configOutputPath = "/opt/aws/amazon-cloudwatch-agent/bin/config.json"
+	agentRuntime = 20 //minutes
```

Comment:
I think that rather than doing ` // minutes` as a comment, we could update the variable name to reflect that the units is minutes. But trying to reason what to change the name to, it feels more awkward than leaving this. Idk. I'll think it over but not something I would block review on.